rustic-doc.el

```diff
@@ -287,14 +288,17 @@ If the user has not visited a project, returns the main doc directory."
          (proc (let ((process-connection-type nil))
                  (apply #'start-process name buf program program-args))))
     (set-process-sentinel
-     proc (lambda (proc _event)
+     proc (lambda (proc event)
             (let ((buf (process-buffer proc)))
-              (when finish-func
-                (funcall finish-func proc))
-              (when (buffer-live-p buf)
-                (kill-buffer buf)))))
+              (if (string-match-p (regexp-quote "abnormally") event)
+                  (message "Could not finish process: %s. See *Messages* or a *rustic-*...* buffer for more info." event)
```

Comment:
> or a *rustic-*...*

You mean the process buffer ?

Reply:
Right, I wasn't sure what it should be called but it seems like it's not worse than what's there now